PLUB/Sources/Views/Meeting/MeetingViewController.swift

```diff
@@ -219,15 +222,41 @@ extension MeetingViewController {
 }
 
 extension MeetingViewController: MeetingCollectionViewCellDelegate {
-  func didTappedSettingButton() {
+  func didTappedSettingButton(plubbingID: Int) {
   }
   
-  func didTappedExitButton() {
+  func didTappedExitButton(plubbingID: Int) {
+    let alert = CustomAlertView(
+      AlertModel(
+        title: "모임에서 나가기\n하시겠어요?",
+        message: nil,
+        cancelButton: "취소",
+        confirmButton: "네, 할게요",
+        height: 210
+      )
+    ) { [weak self] in
+      guard let self = self else { return }
+      self.viewModel.exitMeeting(plubbingID: plubbingID)
```

Comment:
Swift 5.7부터는 let '변수명'만 해도 언래핑이 가능합니다!
```suggestion
      guard let self else { return }
      self.viewModel.exitMeeting(plubbingID: plubbingID)
```

Swift 5.8(Xcode 14.3)부터는 self 참조를 생략할 수 있습니다.

```suggestion
      guard let self else { return }
      viewModel.exitMeeting(plubbingID: plubbingID)
```

Reply:
> 아래는 변경안하셔도 무방합니다!
> 
> 그리고 iPhone SE에서 내모임 사이즈가 커서 아래 인디케이터가 간신히 보여요 ㅜ 나중에 수정 가능하실까용🥹 <img alt="" width="30%" src="https://user-images.githubusercontent.com/57972338/232393045-29554fe3-cb86-4f07-ae92-2db50099c8a1.png">

넵 이거 사이즈가 아니라 비율로 대응해줘야할 것 같아서 다음 pr에 반영해서 올리겠습니다!